src/saftvrqmie/dft/dispersion.rs

```diff
@@ -0,0 +1,105 @@
+use crate::saftvrqmie::eos::dispersion::{dispersion_energy_density, Alpha};
+use crate::saftvrqmie::parameters::SaftVRQMieParameters;
+use feos_core::EosResult;
+use feos_dft::{
+    FunctionalContributionDual, WeightFunction, WeightFunctionInfo, WeightFunctionShape,
+};
+use ndarray::*;
+use num_dual::DualNum;
+use std::fmt;
+use std::sync::Arc;
+
+/// psi Parameter for DFT (Sauer2017)
+const PSI_DFT: f64 = 1.3862; // 1.475
```

Comment:
Yes, the PC-SAFT parameters works well with the SAFT-VRQ Mie EoS.

Reply:
Morten I renamed the binary parameter file to `aasen2020_binary.json` and linked the publication in the readme (since those parameters were used with `hammer2023.json`). Is that change Ok for you?

Reply:
Great! Thanks!